Lib/test/test_dis.py

```diff
@@ -219,6 +220,22 @@ def bug42562():
        RETURN_VALUE
 """
 
+# [255, 255, 255, 252] is -4 in a 4 byte signed integer
+bug46724 = bytes([
+    144, 255, # EXTENDED_ARG
```

Comment:
Why not write `opcode.EXTENDED_ARG` and `opcode.opmap["JUMP_FORWARD"]` so we don't rely on the specific numeric values?

Reply:
Sounds good, updated